content/python/concepts/file-methods/terms/remove/remove.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.remove()'
+Description: 'Allows the user to delete a file if it exists.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Files'
+  - 'Functions'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The Python `.remove()` file method allows the user to delete a file if it exists in the system or computer. [`.unlink()`](https://www.codecademy.com/resources/docs/python/file-methods/unlink) is an alternative to delete a file.
+
+**Note**: A `FileNotFoundError` is raised if the file is not found or isn't in the current directory.
+
+## Syntax
+
+This method is provided by the `os` module which must be imported beforehand.
+
+```pseudo
+os.remove("file_name.file_type")
```

Comment:
```suggestion
import os

os.remove("file_name.file_type")
```

Given the sentence in line # 23 should we include `import os` as suggested above?